bebraven/platform
# Fix updating video URL input clicking between iframes

**Task**: https://app.asana.com/0/1170776727341290/1174986069483581

**Test**

- 2 new sections, add 2 iframes
- Change URL on one
- Click between two, verify URL input updates

<img width="1202" alt="Screen Shot 2020-05-08 at 2 45 43 PM" src="https://user-images.githubusercontent.com/62911934/81452226-349ca300-913b-11ea-8550-924f0ea4a786.png">
<img width="1196" alt="Screen Shot 2020-05-08 at 2 46 05 PM" src="https://user-images.githubusercontent.com/62911934/81452227-35353980-913b-11ea-80a9-a08151e3d957.png">
